### PR TITLE
option to remember invoice type choice

### LINKED ISSUE #396: Finish styling QR format chooser
https://www.figma.com/file/jv72zrIZkNPz70qMsau0K9/Mutiny?type=design&node-id=1798-18213&mode=dev

Might want to make a whole different "styledradiogroup" component to cut down on prop complexity

Also should add a "remember my choice for next time" checkbox

### PR DESCRIPTION
**Pull request by futurepaul**
closes #396

### CODE REVIEW

**Review by TonyGiorgio — APPROVED** [review]

File: src/components/layout/Radio.tsx [hunk 1]
```diff
@@ -10,18 +12,31 @@ type Choices = {
...
     accent?: "red" | "white";
     vertical?: boolean;
+    delayOnChange?: boolean;
 }) {
+    const [value, setValue] = createSignal(props.initialValue);
+
+    async function onChange(value: string) {
+        setValue(value);
+
+        // This is so if the modal is going to be closed after value change, it will show the choice briefly first
```
Comment (benthecarman) [inline src/components/layout/Radio.tsx:28]: Good add, it was a bit abrupt before

**Review by benthecarman — APPROVED** [review]

File: src/components/layout/Radio.tsx [hunk 2]
```diff
@@ -10,18 +12,31 @@ type Choices = {
 
 // TODO: how could would it be if we could just pass the estimated fees in here?
 export function StyledRadioGroup(props: {
-    value: string;
+    initialValue: string;
```
[the remark this answers is not in the document]
Comment (futurepaul) [reply src/components/layout/Radio.tsx:15]: this is a generic radio component we're also going to use it for fee selection (soon TM)

**Comment by futurepaul** [comment]
> It's a bit weird now when you make a selection that it goes away so you can't hit the remember button. However, I'm guessing that 90 like 90% of the time you aren't going to be changing your preferred type so maybe that's fine.

yeah it felt like a compromise but for the use case where you're switching often I didn't want to re-add the friction of a confirm

**Comment by futurepaul** [comment]
gonna merge for now but we can keep iterating

**Pull request merged**